nemo/setup.py

```diff
@@ -19,7 +19,7 @@
         "License :: OSI Approved :: Apache Software License"
     ],
     install_requires=[
-        'torch>=1.2.0,<1.4',
```

Comment:
I would just use 'torch' - that seemed to work well in terms of not breaking anything. Safer.

Reply:
removed version check